src/components/Dropdown/Dropdown.tsx

```diff
@@ -85,37 +90,53 @@ export function Dropdown(props: PropsWithChildren<DropdownProps>): JSX.Element {
       setLastTypedOrSubmittedValue(parentQuery);
       updateFocusedItem(-1, parentQuery);
     }
-  }, [parentQuery, lastTypedOrSubmittedValue, updateFocusedItem, setLastTypedOrSubmittedValue]);
+  }, [
+    parentQuery,
+    lastTypedOrSubmittedValue,
+    updateFocusedItem,
+    setLastTypedOrSubmittedValue
+  ]);
 
   useRootClose(containerRef, () => {
     toggleDropdown(false);
   }, { disabled: !isActive });
 
   useGlobalListener('keydown', e => {
-    if (e.key === 'ArrowDown' || e.key === 'ArrowUp') {
-      e.preventDefault();
+    if (e.key === 'Tab' || e.key === 'ArrowDown' || e.key === 'ArrowUp') {
```

Comment:
maybe I'm missing something, but what does this `if` statement add?

Reply:
oh whoops that was also from when i was experimenting